smelter-core/src/pipeline/decklink/capture.rs

```diff
@@ -136,7 +137,7 @@ impl ChannelCallbackAdapter {
         data: bytes::Bytes,
         pts: Duration,
     ) -> Frame {
-        let data = if width != bytes_per_row * 2 {
+        let data = if width * 2 != bytes_per_row {
```

Comment:
This was a bug that disable performance optimization, but did not break anything